pandas/tests/window/test_grouper.py

```diff
@@ -297,6 +297,32 @@ def test_groupby_rolling_center_center(self):
         )
         tm.assert_frame_equal(result, expected)
 
+    @pytest.mark.parametrize("min_periods", [5, 4, 3])
+    def test_groupby_rolling_center_min_periods(self, min_periods):
+        df = pd.DataFrame({"group": ["A"] * 10 + ["B"] * 10, "data": range(20)})
```

Comment:
can you add the approrpaite issues numbers as a comment

Reply:
e.g `# GH 36040`